packages/web-react/src/components/IconBox/README.md

```diff
@@ -0,0 +1,79 @@
+# IconBox
+
+The `IconBox` component is a composition of the `Icon` inside the `Box` component.
+
+```jsx
+<Icon name="search" />
+```
+
+## Border Radius
+
+You can define border radius `borderRadius` prop.
+
+```jsx
+<Icon name="search" borderRadius="circle" />
+```
+
+## Has Border
+
+You can set `hasBorder` prop to render the `IconBox` without a border.
+
+```jsx
+<Icon name="search" hasBorder={false} />
+```
+
+## Size
+
+You can define size using the `size` prop.
+
+```jsx
+<Icon name="search" size="large" />
+```
+
+## Color
+
+You can define color using the `color` prop.
```

Comment:
Could you be more descriptive about what exactly this prop does? It sets the color of three different properties: border color, background color and content color (for vector paths).